### ISSUE TITLE
How to train the RoseTTAFold

### ISSUE THREAD

**Issue by wuwo007** [RosettaCommons/RoseTTAFold#84]
Thanks for sharing this exciting work!
Nowdays, I want to train the RoseTTAFold on my own dataset. However, I noticed that there was no loss function. Would you like to provide some instructions or tutorials? Thanks very much!

**Comment by wmustafaAwad** [comment]
Hi @wuwo007  , the authors do not provide the training code. I assume this is because they are licensing the training weights for a small fee for commercial use.<br/>
The way you can go about this is by implementing a pytorch training loop for the model, including the loss function. These are both non-trivial in terms of deep learning coding skills (pytorch library). Good news is, however, that the authors have detailed their loss function in their [supplementary materials pdf (page 6)](https://www.biorxiv.org/content/biorxiv/early/2021/06/15/2021.06.14.448402/DC1/embed/media-1.pdf)

**Comment by esun0087** [comment]
hi @wuwo007  , I am trying to train rosettafold and is confused about something in the network , we can discuss .  how can i connect you

**Comment by wuwo007** [comment]
> hi @wuwo007 , I am trying to train rosettafold and is confused about something in the network , we can discuss . how can i connect you

We can communicate with email.  My email is gao_zhi_qiang@126.com.

**Comment by esun0087** [comment]
> > hi @wuwo007 , I am trying to train rosettafold and is confused about something in the network , we can discuss . how can i connect you
> 
> We can communicate with email. My email is [gao_zhi_qiang@126.com](mailto:gao_zhi_qiang@126.com).

i have send email to u
